src/common/sync-box/create-sync-box.injectable.ts

```diff
@@ -17,7 +17,7 @@ const createSyncBoxInjectable = getInjectable({
     const sendToAgnosticChannel = di.inject(sendToAgnosticChannelInjectionToken);
     const getSyncBoxState = (id: string) => di.inject(syncBoxStateInjectable, id);
 
-    return (id: string): SyncBox<any> => {
+    return <TData>(id: string): SyncBox<TData> => {
```

Comment:
Why is it possible to create a sync box without a default value?

Reply:
At the moment default values are not welcome, since that's very complex topic. Main process is always responsible from setting the initial value.

Reply:
Hmm. Better to use ´| undefined´ only when setting no value is intentional. Adding defensive code against attempts to get value before initial value has arrived.